leits/MeetingBar
# Add setting to only show events starting in x minutes.

### Status
**READY**

### Description
This PR adds a setting to only show events in x minutes. It implements the feature request #176. It's only filtering out future events that start later then the configured threshold. Current or past events are displayed as configured by other settings.

![Bildschirmfoto 2021-02-12 um 19 08 41](https://user-images.githubusercontent.com/1190948/107805499-baeab300-6d65-11eb-89d5-20d8ca377e2b.png)


### Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. Have the next event in e.g. 60 minutes.
2. Open Settings and configure it to show only events starting in 30 minutes.

The event should not be displayed in the title.

3. Move event to in 25 minutes.

The event should be now displayed in the title.
